pages.ts

```diff
@@ -83,3 +94,140 @@ export async function loadData(
 
   return { ...pageData, components: componentsWithData };
 }
+
+export const fetchPageFromPathname = async (
+  path: string,
+  siteId: number
+): Promise<PageWithParams | null> => {
+  const { data: pages, error } = await getSupabaseClient()
+    .from("pages")
+    .select("id, name, data, path")
+    .match({ site: siteId, archived: false })
+    .is("flag", null)
+    .is("archived", false);
+
+  const routes = pages?.map((page) => ({
+    page,
+    pattern: page.path,
+  })) as Array<{ pattern: string; page: Page }>;
+
+  sortRoutes(routes);
```

Comment:
Allows that specific paths have priority over dynamic ones 


<img width="702" alt="Screen Shot 2022-11-02 at 23 03 01" src="https://user-images.githubusercontent.com/18706156/199640354-04385d24-0150-402d-91dc-29449682f556.png">